xml/System/Uri.xml

```diff
@@ -2394,27 +2394,25 @@ If you used an escaped string to construct this instance (for example, `"http://
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- The <xref:System.Uri.GetLeftPart%2A> method returns a string containing the leftmost portion of the URI string, ending with the portion specified by `part`.
 
- <xref:System.Uri.GetLeftPart%2A> includes delimiters in the following cases:
+The <xref:System.Uri.GetLeftPart%2A> method returns a string containing the leftmost portion of the URI string, ending with the portion specified by `part`.
 
--   <xref:System.UriPartial.Scheme> includes the scheme delimiter.
+<xref:System.Uri.GetLeftPart%2A> includes delimiters in the following cases:
 
--   <xref:System.UriPartial.Authority> does not include the path delimiter.
+- <xref:System.UriPartial.Scheme> includes the scheme delimiter.
+- <xref:System.UriPartial.Authority> does not include the path delimiter.
+- <xref:System.UriPartial.Path> includes any delimiters in the original URI up to the query or fragment delimiter.
```

Comment:
```suggestion
- <xref:System.UriPartial.Path> includes the path delimiter and any delimiters in the original URI up to the query or fragment delimiter.
```

E.g. For `http://foo`, `UriPartial.Path` will insert the path delimiter `http://foo/`